src/lib/y2storage/filesystems/type.rb

```diff
@@ -30,6 +30,82 @@ class Type
       include StorageEnumWrapper
 
       wrap_enum "FsType"
+
+      FILESYSTEMS = {
+        btrfs:    {
+          name: "BtrFS"
+        },
+        ext2:     {
+          name: "Ext2"
+        },
+        ext3:     {
+          name: "Ext3"
+        },
+        ext4:     {
+          name: "Ext4"
+        },
+        hfs:      {
+          name: "MacHFS"
+        },
+        hfsplus:  {
+          name: "MacHFS+"
+        },
+        jfs:      {
+          name: "JFS"
+        },
+        msdos:    {
+          name: "FAT"
+        },
+        nilfs2:   {
+          name: "NilFS"
+        },
+        ntfs:     {
+          name: "NTFS"
+        },
+        reiserfs: {
+          name: "Reiser"
+        },
+        swap:     {
+          name: "Swap"
+        },
+        vfat:     {
+          name: "FAT"
+        },
+        xfs:      {
+          name: "XFS"
+        },
+        iso9669:  {
+          name: "ISO9660"
+        },
+        udf:      {
+          name: "UDF"
+        }
+      }
+
+      ROOT_FILESYSTEMS = [:ext2, :ext3, :ext4, :btrfs, :reiserfs, :xfs]
+
+      HOME_FILESYSTEMS = [:ext2, :ext3, :ext4, :btrfs, :reiserfs, :xfs]
+
+      # Allowed filesystems for root
+      #
+      # @return [Array<Filesystems::Type>]
+      def self.root_filesystems
+        ROOT_FILESYSTEMS.map { |f| find(f) }
+      end
+
+      # Allowed filesystems for home
+      #
+      # @return [Array<Filesystems::Type>]
+      def self.home_filesystems
+        HOME_FILESYSTEMS.map { |f| find(f) }
+      end
+
+      # Human readable text for a filesystem
+      #
+      # @return [String]
+      def to_human
+        FILESYSTEMS[to_sym][:name]
```

Comment:
Doesn't look robust. What if a new type is added in libstorage but we don't add the corresponding name?